.spelling

```diff
@@ -113,6 +113,7 @@ filesystem
 firefox-data-docs
 FirefoxReality
 FireTV
+frac
```

Comment:
This is to get around the fix that mdspell checks the spelling on MathJax blocks 🙄